stable/postgresql/Chart.yaml

```
@@ -1,5 +1,5 @@
 name: postgresql
-version: 3.9.5
+version: 3.10.0
```

please bump to 4.0.0, this might be a breaking change

```suggestion
version: 4.0.0
```